frontend/src/metabase/pulse/components/PulseEditCards.jsx

```diff
@@ -61,7 +68,7 @@ export default class PulseEditCards extends Component {
         if (hasAttachment) {
             notices.push({
                 head: t`Attachment`,
-                body: <AttachmentWidget card={card} onChange={(card) => this.setCard(index, card)} />
+                body: <AttachmentWidget card={card} onChange={(card) => this.setCard(index, card)} trackPulseEvent={this.trackPulseEvent} />
```

Comment:
this makes me itchy.

Reply:
In order to get it under the proper "PulseEdit" vs "PulseCreate" category I either had to pass the pulseId or this. I opted for this since at least this way I didn't have to replicate the "PulseEdit" "PulseCreate" ternary inside of `AttachmentWidget.`

Reply:
(I don't really like it either for the record).

Reply:
Do we actually need separate categories for `PulseEdit` and `PulseCreate`?